core-common/src/main/java8/org/glassfish/jersey/internal/jsr166/SubmissionPublisher.java

```diff
@@ -1,3 +1,19 @@
+/*
```

Comment:
This file has CC0 license. We must not put EPL in this file.

Reply:
I have removed the copy right and added this path for exclusion:
/core-common/src/main/java8/org/glassfish/jersey/internal/jsr166